docs/reference/cli/kcl/fmt.md

```diff
@@ -4,11 +4,11 @@ sidebar_position: 2
 
 # Format
 
-KCL supports formatting multiple KCL files via built-in command line tools. This article demonstrates the KCL coding style and how to use the KCL formatting tool.
+KCL supports formatting multiple KCL files via the built-in formatting tool. This article demonstrates the KCL code style and how to use the KCL Format tool.
 
 ## Code Style
 
-The KCL formatting tool modifies the files according to the KCL coding style: [Style Guide for KCL Code](../../lang/lang/spec/codestyle.md)
+The KCL Format tool modifies the files according to the KCL code style: [Style Guide for KCL Code](../../lang/lang/spec/codestyle.md)
```

Comment:
The KCL Fmt tool or The KCL Format tool

since the KCL document generating tool is stated as "The KCL Docgen tool", better to use consistent pattern